internal/fs/inode/file.go

```diff
@@ -397,13 +397,6 @@ func (f *FileInode) Attributes(
 	attrs.Atime = attrs.Mtime
 	attrs.Ctime = attrs.Mtime
 
-	// Clobbered check makes a stat call to GCS. Avoiding stat calls to GCS for
-	// local files when fetching attributes. Any validations will be done during
-	// sync call.
-	if f.IsLocal() {
```

Comment:
I didn't understand, why are we removing this?

Reply:
Earlier i thought of removing call to GCS for local files. But now it is required to check if the file is clobbered or not to ensure Nlink is set correctly.